examples/persistence_transform.py

```diff
@@ -0,0 +1,52 @@
+"""
+========================================
+Persistence Transform in MapSequence
+========================================
+Persistence Transform is a simple image processing technique that is useful for the visualization
+and depiction of gradually evolving structures.
+This example illustrates persistence transform using the AIA_193_CUTOUT series.
+
+.. note::
+   This example requires ``mpl_animators`` to be installed.
+"""
+
+import matplotlib.cm as cm
+import matplotlib.colors as colors
+import matplotlib.pyplot as plt
```

Comment:
```suggestion
from matplotlib import cm
from matplotlib import colors
import matplotlib.pyplot as plt
```
Do these changed imports work?

Reply:
Yea will change if it runs